nodejs/eks/cluster.ts

```diff
@@ -312,6 +359,12 @@ export interface ClusterOptions {
      */
     instanceRole?: pulumi.Input<aws.iam.Role>;
 
+    /**
+     * The array of instance role ARNs to map into the aws-auth ConfigMap. This
+     * is a required step to specify a NodePool's instanceRoleArn.
+     */
+    instanceRoleArns?: pulumi.Input<aws.ARN>[];
```

Comment:
Per above - I think a lot of this is simpler if we just do `instanceRoles?`?

Also - should add notes to both this and `instanceRole` that they are mutually exclusive.  As well as checks when reading from these that enforce and throw if both are provided.